addons/godot-xr-tools/objects/Viewport_2D_in_3D.gd

```diff
@@ -8,7 +8,7 @@ export var enabled = true setget set_enabled
 export var screen_size = Vector2(3.0, 2.0) setget set_screen_size
 export var viewport_size = Vector2(300.0, 200.0) setget set_viewport_size
 export var transparent = true setget set_transparent
-export (PackedScene) var scene = null setget set_scene, get_scene
+export var scene: PackedScene = null setget set_scene, get_scene
```

Comment:
I wonder if this should be ` = PackedScene()` instead of ` = null` or leaving the assignment out all together as the default is an empty object.

Reply:
I removed the null assignment as gdscript is going to do a better job of picking a default/empty value.